addons/survey/views/survey_templates.xml

```diff
@@ -283,7 +283,7 @@
                         </t>
                         <t t-call="survey.survey_button_retake"/>
                         <div t-if="survey.scoring_type != 'scoring_without_answers'">
-                            If you wish, you can <a t-att-href="'/survey/print/%s?answer_token=%s&amp;review=True' % (survey.access_token, answer.access_token)">review your answers</a>
+                            If you wish, you can <a t-att-href="'/survey/print/%s?answer_token=%s&amp;review=True' % (survey.access_token, answer.access_token)" id="review_answers">review your answers</a>
```

Comment:
Same here, maybe you could target "a" ?

Not sure, there may be multiple links in the page, to double-check.
If it's the case it's better to introduce a class than an ID.